packages/reakit/src/Table/Table.md

```diff
@@ -3,73 +3,79 @@ It applies a basic styling to its children to improve differentiation between he
 Table renders by default as a `<table>`.
 
 ```jsx
-import { Block } from "reakit";
+import { Block, Provider } from "reakit";
+import themeDefault from "reakit-theme-default";
 
-<Block overflowX="auto">
-  <Table>
-    <caption>A Basic table</caption>
-    <thead>
-      <tr>
-        <th/>
-        <th>Chars</th>
-        <th>Popularity</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <th>Foo</th>
-        <td>3</td>
-        <td>0.7</td>
-      </tr>
-      <tr>
-        <th>Bar</th>
-        <td>3</td>
-        <td>0.4</td>
-      </tr>
-    </tbody>
-  </Table>
-</Block>
+<Provider theme={themeDefault}>
```

Comment:
We don't want `Provider` in our examples. It should be added automatically in the Playground and we should provide a way for the user change the theme.